<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClusterOptions</class>
 <widget class="QDialog" name="ClusterOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cluster Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="clusterGroup">
         <property name="text">
          <string>Cluster</string>
         </property>
         <property name="buddy">
          <cstring>clusterBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="clusterBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="NewName">
         <property name="text">
          <string>NewName</string>
         </property>
         <property name="buddy">
          <cstring>NewNameEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="NewNameEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="colorGroup">
           <property name="text">
            <string>Color</string>
           </property>
           <property name="buddy">
            <cstring>redSpin</cstring>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="red">
               <property name="text">
                <string>Red</string>
               </property>
               <property name="buddy">
                <cstring>redSpin</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="redSpin">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="green">
               <property name="text">
                <string>Green</string>
               </property>
               <property name="buddy">
                <cstring>greenSpin</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="greenSpin">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="blue">
               <property name="text">
                <string>Blue</string>
               </property>
               <property name="buddy">
                <cstring>blueSpin</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="blueSpin">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="pointSize">
           <property name="text">
            <string>Point Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="pointSizeSpinBox">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
